engines/postgres.py

```diff
@@ -96,8 +96,6 @@ def insert_data_from_file(self, filename):
     def insert_statement(self, values):
         """Returns a SQL statement to insert a set of values"""
         statement = Engine.insert_statement(self, values)
-        if isinstance(statement, basestring):
-            statement = statement.decode("utf-8", "ignore")
```

Comment:
`basestring` is a parent of `str` and `unicode` in python 2. There is no `basestring` in Python 3, and the strings are `unicode` by default. 
Since the `decode()` function converts `bytes` to `str`, and the `str` are already unicode as I import `builtins.str` at the beginning, that means that I should've replaced `basestring` with `bytes` here.